wpeverest/everest-forms
# Feature/required field message

### All Submissions:






### Changes proposed in this Pull Request:


Admin can set error message to show, for individual required field. Currently, this feature is only applied to the following fields, which are `General Fields` and some `Advanced Fields`:
1. First Name (first-name)
2. Last Name (last-name)
3. Single Line Text (text)
4. Paragraph Text (textarea)
5. Dropdown (select)
6. Multiple Choice (radio)
7. Checkboxes (checkbox)
8. Number (number)
9. Email (email)
10. Website / URL (url)

### How to test the changes in this Pull Request:

1. Create a new Everest Form
2. Add some(or all) of the above mentioned fields
3. Change it to required
4. Set required message for each field. A field to set the message appears right below the `required` checkbox
5. Save and go to preview page
6. Try to submit the form without entering any value in the required fields. The error messages will appear below the required fields.

### Types of changes:




### Other information:




### Changelog entry

> Feature: Editable error message for individual required field.
